lib/src/client.dart

```diff
@@ -1312,6 +1312,9 @@ class Client extends MatrixApi {
 
   final CachedStreamController<Event> onGroupMember = CachedStreamController();
 
+  final CachedStreamController<String> onDeleteLocalEvent =
```

Comment:
now that the method is called `cancelSend`, we should also use the same terminology everywhere. This could be called `onCancelEventSend` or something?